src/test/java/org/broadinstitute/hellbender/utils/bigquery/AvroFileReaderUnitTest.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.hellbender.utils.bigquery;
+
+
+import org.apache.avro.generic.GenericRecord;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.apache.avro.Schema;
+
+import java.util.*;
+
+/**
+ * A class to test the functionality of {@link AvroFileReader}.
+ */
+public class AvroFileReaderUnitTest extends GATKBaseTest {
+
+    private static final String avroFileURI = "gs://broad-dsp-spec-ops/kcibul/acmg.35.chr20.1mb.cohort.extract.avro";
+    private static final AvroFileReader AvroFile = new AvroFileReader(avroFileURI);
```

Comment:
java naming conventions would be `avroFile` for the variable name.  There's a bunch of these in here